pixart.js

```diff
@@ -1 +1,35 @@
+// var button = document.querySelector("#set-color");
+// var colors = document.querySelector("#color-field");
+// var brusher = document.querySelector('.brush');
+//
+//
+// form.addEventListener('submit', function(event){
+//   event.preventDefault();
+//   brusher.style.backgroundColor = colors.value;
+// });
+//
+// for( var i = 0; i < 5000; i++){
+//   var div = document.createElement('div');
+//   div.classList.add('square');
+//   div.addEventListener('mouseover', function(){
+//     this.style.backgroundColor = brusher.style.backgroundColor;
+//   });
+//   document.body.appendChild(div);
+// }
 
+//Since I am using Jquery I don't need the variables.
+//First I need to target the button.
+$("#form")[0].addEventListener("submit",function(pizza){
+  event.preventDefault()//This took a while to figure out. I can use this for the cash register exercise.
```

Comment:
so, i'm a bit suprised this work. Becareful about your argument. Make sure that `pizza` is `event` instead. we can only call `preventDefault()` on an event object. So in your code it should actually be written:

```js
pizza.preventDefault()
```

The convention we follow as developers is `event`